packages/airnode-validator/src/api/api.ts

```diff
@@ -53,10 +53,14 @@ export function parseReceipt(receipt: unknown): ValidationResult<Receipt> {
 const NO_MATCH_REGEXP = /($^)/;
 // Regular expression matching ES template literal delimiter (${}) with escaping
 // https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L175
-const ES_MATCH_REGEXP = /\$\{([^\\}]*(?:\\.[^\\}]*)*)\}/g;
+const ES_MATCH_REGEXP = /(?<!\\)\$\{([^\\}]*(?:\\.[^\\}]*)*)\}/g;
+// Regular expression matching the escaped ES template literal delimiter (${}). We need to use "\\\\" (four backslashes)
```

Comment:
:exploding_head: I struggled with this, but didn't spend too much time on it. Nice.